MIGRATION.md

```diff
@@ -32,6 +32,44 @@ import {
 } from '@sentry/minimal';
 ```
 
+## Explicit Client Options
+
+In v7, we've updated the `Client` to have options seperate from the options passed into `Sentry.init`. This means that constructing a client now requires 3 options: `integrations`, `transport` and `stackParser`. These can be customized as you see fit.
+
+```ts
+import { BrowserClient, defaultStackParsers, defaultIntegrations, makeFetchTransport } from '@sentry/browser';
+import { stackParserFromOptions } from '@sentry/utils';
+
+// New in v7:
+const client = BrowserClient({
```

Comment:
I would be careful using `PascalCase` naming when the intention is doing a function call.

It is easy to miss that they are technically functions and assume that they are constructors and people adding `new BrowserClient(...)`. And the worst is that it is "technically correct" from the perspective of the runtime (but wrong based on the usage, and intention).

I will recommend to do `makeBrowserClient` or `browserClient` or `newBrowserClient` but avoid `PascalCase`.

Reply:
yup, this is my bad. `BrowserClient` is still a class for now - so there should be the `new` keyword here - we'll maybe revisit this at a later time though,